### PR TITLE
DEV-5311: Submission Type section, merging Periods 1+2

### PR DESCRIPTION
**Pull request by dpb-bah**
**High level description:**

* Added "Submission Type" section when creating a submission.
* Updated list of selectable periods. 

**Technical details:**

* Test submissions will skip the check for published_submission_ids
* Test submissions will include `test_submission: true` when uploading files.
* Periods 1 and 2 have been merged.
    * Period 2 gets sent back for the `published_submission_ids` check.
    * `upload_dabs_files` gets the start of period 1 and end of period 2
* Period options have been reformatted for clarity.

**Link to JIRA Ticket:**

[DEV-5311](https://federal-spending-transparency.atlassian.net/browse/DEV-5311)

**Mockup**
https://bahdigital.invisionapp.com/share/7SIADUOQP4K#/screens/296037245

The following are ALL required for the PR to be merged:

Author: 
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled Demo including Design/Testing/Front-end OR Provided Instructions for Local Testing above and in JIRA
- [x] Verified cross-browser compatibility
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)

Reviewer(s):
- [x] Design review completed
- [ ] Frontend review completed
- [ ] Merged concurrently with [Backend#1234](https://github.com/fedspendingtransparency/data-act-broker-backend/pull/1234)

### CODE REVIEW

File: src/js/components/addData/metadata/SubmissionTypeField.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                </div>
+                <div className="row">
+                    <div className="col-sm-12 pos-rel text-left usa-da-submissiontype">
+                        <div className="usa-da-submissiontype-group">
+                            <input
+                                type="radio"
+                                id="usa-da-submisisontype-test"
+                                name="submisisontype"
+                                value="certifiable"
+                                onClick={this.pickedType.bind(this, 'test')}
```
Comment (alburde1) [inline src/js/components/addData/metadata/SubmissionTypeField.jsx:50]: Can these binds be made in the constructor? Also, the value is flipped from the onClick

File: src/_scss/pages/addData/_content.scss [hunk 2]
```diff
@@ -18,6 +18,14 @@
                 }
             }
         }
+        .usa-da-submissiontype {
+            .usa-da-submissiontype-group {
```
Comment (alburde1) [inline src/_scss/pages/addData/_content.scss:22]: While I'm here, why is `submissiontype` one word?

Comment (dpb-bah) [reply src/_scss/pages/addData/_content.scss:22]: `.usa-da-datetype` above was one so i kept it consistent

Comment (alburde1) [reply src/_scss/pages/addData/_content.scss:22]: It's weird that it's one word when it's clearly 2 words. At least update the one we made. If you want to clean up you can do the other one, not required.

**Pull request merged**